### ISSUE TITLE
Chapter 7: add 1565 floppy drive to the tables

### ISSUE THREAD

**Issue by MauriceMEGA65** [MEGA65/mega65-user-guide#335]
To have the tables represent the screenshot, please add 1565 floppy drive.
I believe CBM would have allocated it to UNIT 9?

![Screenshot 2021-10-13 at 22 20 12](https://user-images.githubusercontent.com/75681098/137207486-21e4ee25-dbb9-4d87-85b3-d9f82dfdaa8f.jpg)

**Comment by Impakt** [comment]
I'll do whatever you guys want, but I had never heard of the 1565, so not sure how useful it'll be to the average COmmodore fan/user

**Comment by MauriceMEGA65** [comment]
I actually like to keep it in as there is info available on the web: http://www.zimmers.net/cbmpics/d1565.html

**Comment by Impakt** [comment]
I had to add a page break otherwise the footnotes ended up on the next page. I either need to pad it out so the table moves to the next page, or cut content out. 

The problem is the table sits right at the bottom of page as it is.

![Screen Shot 2021-10-15 at 12 01 15 am](https://user-images.githubusercontent.com/1773489/137322345-98773d42-9c54-4382-b788-dbe4e1008c5a.png)

**Comment by MauriceMEGA65** [comment]
Yes, keep "The 1565 ........ to production", but remove the text after

**Comment by MauriceMEGA65** [comment]
I liked your proposal: 

![Screenshot 2021-10-16 at 14 34 10](https://user-images.githubusercontent.com/75681098/137587620-de254890-ce08-436e-a3b9-853fe1b4a687.jpg)
